syne_tune/optimizer/schedulers/pbt.py

```diff
@@ -34,32 +31,7 @@ class PBTTrialState:
     stopped: bool = False
 
 
-_ARGUMENT_KEYS = {
-    "resource_attr",
-    "population_size",
-    "perturbation_interval",
-    "quantile_fraction",
-    "resample_probability",
-}
-
-_DEFAULT_OPTIONS = {
-    "resource_attr": "time_total_s",
-    "population_size": 4,
-    "perturbation_interval": 60.0,
-    "quantile_fraction": 0.25,
-    "resample_probability": 0.25,
-}
-
-_CONSTRAINTS = {
-    "resource_attr": DA_String(),
-    "population_size": DA_Integer(1, None),
-    "perturbation_interval": DA_Float(0.01, None),
-    "quantile_fraction": DA_Float(0.0, 0.5),
-    "resample_probability": DA_Float(0.0, 1.0),
-}
-
-
-class PopulationBasedTraining(FIFOScheduler):
+class PopulationBasedTraining(TrialScheduler):
```

Comment:
Could we add test for this one? (potentially dropping the tests for the other?)
I checked and I could not see any test for this one.

Reply:
Its all good actually, I missed the test in my review.